<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QDialog" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/elements/battle.jpg);</string>
  </property>
  <widget class="QPushButton" name="lvUpButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/battle.png);
background: transparent;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lvUpBatteLablel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 96pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>升级赛</string>
   </property>
  </widget>
  <widget class="QLabel" name="fightBattleLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;
font: 700 96pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>决斗赛</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fightBattleButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>-40</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/battle.png);
background: transparent;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1189</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/elements/close.png);
background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
